pkg/cli/debug/debug.go

```diff
@@ -375,11 +376,52 @@ func (o *DebugOptions) RunDebug() error {
 	if err != nil {
 		klog.V(4).Infof("Unable to get exact template, but continuing with fallback: %v", err)
 	}
+	var debugNodeNS *corev1.Namespace
+	generateName := names.SimpleNameGenerator.GenerateName("openshift-debug-node-")
+	// create namespace only if debugging a node
+	if o.IsNode && len(o.ToNamespace) == 0 && len(o.Namespace) == 0 {
+		if !o.Attach.Quiet {
+			fmt.Fprintf(o.Attach.ErrOut, "\nCreating debug namespace/%s ...\n", generateName)
```

Comment:
`o.ErrOut`

Reply:
Drop the newline at the beginning too

Reply:
updated, and also I changed the above`if len(o.Namespace) == 0`  to `if !o.ExplicitNamespace`  bc that was broken